lib/unit-ruby/util/error.rb

```diff
@@ -1,10 +1,10 @@
 module Unit
   class Error < StandardError
-    attr_accessor :title, :status, :details, :detail
+    attr_accessor :message, :status, :details, :detail
 
     def initialize(api_response)
       error = api_response['errors'].first
-      @title = error['title']
+      @message = error['title']
```

Comment:
New update to the `Unit::Error` interface

Reply:
Aside from the third part libraries not parsing this correctly this change should also help with how the error message is shown in a console. Right now it just shows `Unit::Error Unit::Error`, not very helpful. 

The alternative to this approach would've been to keep the @title attr to match what's coming from unit and the just do something like:

```
def message
    title
end

or

def message
    "title: #{title}, status: ...."
    # or some other formatting to make it more readable
end
```